lib/app/store.js

```diff
@@ -11,12 +11,8 @@ const filenames = files.keys()
 let storeData = {}
 
 // Check if {dir.store}/index.js exists
-let indexFilename
-filenames.forEach((filename) => {
-  if (filename.indexOf('./index.') !== -1) {
-    indexFilename = filename
-  }
-})
+const indexFilename = filenames.find(filename => filename.includes('./index.'))
```

Comment:
The leading dot means we are only getting relative paths, not `/absolute/index.js`, so...
```js
const indexFilename = filenames.find(filename => filename.slice(0,8) === './index.')
```

Reply:
It's enough ;)

Reply:
I'd rather go with `startsWith`

Reply:
This is inline-ponyfilled `.startsWith`, since this is currently not transpiled afaik.

Reply:
`startsWith` has very early support (https://node.green/#ES2015-built-ins-well-known-symbols-Symbol-match--String-prototype-startsWith)